### ISSUE TITLE
Update bugsnag-react-native to v2.6

### ISSUE THREAD

**Issue by rye** [StoDevX/AAO-React-Native#1867]
See #1861 for the problems (on Android) that our first pass at this caused.

Note that in that stack trace I see a lot of complaining about maps.  I wondered if this could be solved by running
- `react-native unlink react-native-maps`, then
- `react-native unlink bugsnag-react-native`

and then re-linking them in reverse order via
- `react-native link bugsnag-react-native`, then
- `react-native link react-native-maps`

No joy on this attempt, unfortunately.

* * *

`bugsnag-react-native` v2.6.0 requires us to add the Google Maven repository, via adding to `android/build.gradle` the following lines:
```gradle
allprojects {
    repositories {
        // [...]
        maven {
            url 'https://maven.google.com'
        }
        // [...]
    }
}
```

I'm logging this so we know what has already been tried.

**Comment by hawkrives** [comment]
@rye I wonder if we could do this now that we've upgraded our targetSdkVersion?

**Comment by rye** [comment]
@hawkrives, good thought.  It's worth experimenting with that.

**Comment by drewvolz** [comment]
Should we close this issue until we're ready to tackle this at another time?

**Comment by hawkrives** [comment]
No

**Comment by hawkrives** [comment]
This has been fixed; we're now on `bugsnag-react-native@2.11`.
